_posts/2025-04-15-debezium-moves-to-ibm.adoc

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title:  "Debezium Team Transitioning to IBM: Continuing Our Upstream Journey"
+date:   2025-04-15
+tags: [ announcement ]
+author: ccranfor
+---
+
+We're writing to share an important update regarding the Debezium team.
+As part of a broader strategic move, the Debezium team, along with Red Hat's Middleware and Integration Engineering and Products teams, will be transitioning to IBM in July 2025.
+
++++<!-- more -->+++
+
+We understand that news of this nature can come as a surprise, and we want to acknowledge the natural apprehension that accompanies change.
+This decision, while unexpected, represents a strategic alignment of Java expertise between Red Hat and IBM, aimed at fostering greater innovation in the middleware space.
+
+Let us be clear: **our commitment to Debezium remains unwavering**.
+
+The upstream-first development model that has been instrumental to Debezium's success will continue under IBM's stewardship.
+Your contributions and involvement in the Debezium project are highly valued and will remain essential.
+This transition will not impact your ability to participate in the project.
+
+IBM is dedicated to maintaining and fostering the open-source neutrality of Debezium.
```

Comment:
Dunno If we want to reinforce this recalling the foundation move.

Reply:
We prefer to avoid making an explicit link between IBM and the foundation. The move to the foundation is about the ownership of the project while the move to IBM is about whom the core developers are employed by, which are two separate concerns.

Reply:
Well with the latest changes from Jiri and the clarification about "Team vs Project" mention the foundation is not necessary. :wink: